custom_components/tuya_local/devices/casdon_td_pro_2.yaml

```diff
@@ -4,6 +4,7 @@ products:
     name: Casdon TD Pro 2
 primary_entity:
   entity: climate
+  translation_only_key: casdon_oven
```

Comment:
I think just "oven" is better, as other ovens are likely to have similar presets.  Even if some are different, it is no problem having extra presets defined in the translations.

Unlike the error codes, these do not have the "E1" prefix tying them strongly to a specific device implementation.

Reply:
I think using an oven is also feasible; I’m just not sure if this list will become particularly large in the future.